config/config.toml

```diff
@@ -16,8 +16,8 @@ write_timeout = 5000000000
 shutdown_timeout = 5000000000
 
 log_location = "logs"
-# options: trace, debug, info, warning, error, fatal, panic
-log_level = "debug"
+# options: trace, debug, info, warn, error, fatal, panic
+log_level = "warn"
```

Comment:
yeah this needs to be refactored. I changed it to make the logs less noisy on CI and I opened #436 because of it
I'll revert this change for now.

I'd like us to have "environment specific configs" that provide log level and maybe some other stuff